eclipse-collections-api/src/main/java/org/eclipse/collections/api/list/MutableList.java

```diff
@@ -61,24 +63,43 @@
     MutableList<T> withoutAll(Iterable<? extends T> elements);
 
     @Override
-    MutableList<T> newEmpty();
+    default MutableList<T> newEmpty()
```

Comment:
Not sure about this. Isn't the method's intent to return the same type as the one being called? If so, this default implementation seems to break that contract. Was this forced by the calls to this.newEmpty() in the other default methods?

Reply:
This is optional. It's just the implementation from AbstractMutableList moved up. I could remove it.

Reply:
I vote to remove.